.travis.yml

```diff
@@ -43,6 +43,7 @@ deploy:
       repo: apache/openwhisk-runtime-nodejs
   - provider: script
     script: "./tools/travis/publish.sh openwhisk 8 nightly && ./tools/travis/publish.sh openwhisk 10 nightly && ./tools/travis/publish.sh openwhisk 12 nightly && ./tools/travis/publish.sh openwhisk 14 nightly"
+    script: "./tools/travis/publish.sh openwhisk nodejs10Action nightly && ./tools/travis/publish.sh openwhisk nodejs12Action nightly && ./tools/travis/publish.sh openwhisk nodejs14Action nightly && ./tools/travis/publish.sh openwhisk typescript37Action nightly"
```

Comment:
I think you wanted to delete line 45 and replace with line 46.  Not have both line 45 and line 46.